### PR TITLE
New issue form

### PR DESCRIPTION
**Pull request by metalefty**
Hi, 

Some people are still reporting issues with 4.x Stable Edition here.  I suggest introducing an [issue form](https://docs.github.com/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms), a new type of issue template. The new issue form clearly distinguishes between descriptive text and the text to be filled by reporters. It is still possible to open an issue in free form when a new report such as a feature request doesn't suit the form.

This form is based on the form I have been using in [xrdp project](https://github.com/neutrinolabs/xrdp/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml) since last year. I believe this will help to guide 4.x users to the correct path and collect detailed information about bug reports for 5.x.

See my fork to see a demo. I hope you like it. I welcome any suggestions or comments.
* https://github.com/metalefty/SoftEtherVPN/issues/new/choose
* https://github.com/metalefty/SoftEtherVPN/issues/new?assignees=&labels=needs-triage&projects=&template=bug_report_or_issue_report.yml

## Screenshots

image

image

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report_or_issue_report.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
+name: Bug Report or Issue Report
+description: File a bug report or an issue report
+labels: "needs-triage"
```
Comment (metalefty) [inline .github/ISSUE_TEMPLATE/bug_report_or_issue_report.yml:3]: @chipitsine 
Issue form requires at least one default label. I choose "needs-triage" for the default label.
Could you add "needs-triage" label?
https://github.com/SoftEtherVPN/SoftEtherVPN/labels?page=1&sort=name-asc

Comment (chipitsine) [reply .github/ISSUE_TEMPLATE/bug_report_or_issue_report.yml:3]: image

Comment (metalefty) [reply .github/ISSUE_TEMPLATE/bug_report_or_issue_report.yml:3]: Thanks!

**Comment by kfcxzaq084** [comment]
https://github.com/kfcxzaq084/SoftEtherVPN

**Pull request merged**